pages/rooms.js

```diff
@@ -0,0 +1,59 @@
+import RoomItem from '../components/RoomItem';
+import { getAllRooms } from '../services/roomService';
+import styled from 'styled-components';
+import Link from 'next/link';
+import Image from 'next/image';
+import backArrow from '../public/backarrow.png';
+
+export async function getStaticProps() {
+  const rooms = await getAllRooms();
+  return {
+    props: {
+      rooms,
+    },
+  };
+}
+
+export default function Rooms({ rooms }) {
+  return (
+    <>
+      <Link href="/" passHref>
+        <BackAnchor>
+          <Image alt="backbutton" src={backArrow} />
+        </BackAnchor>
+      </Link>
+
+      <RoomList>
+        <Header>All Rooms</Header>
+        {rooms.map((room) => {
+          return <RoomItem key={room.id} roomData={room} />;
+        })}
+      </RoomList>
+    </>
+  );
+}
+
+const RoomList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  gap: 51px;
+  /* position: absolute; */
```

Comment:
When creating a PR, it's best to remove all comments beforehand :)

Reply:
Thanks for the hint. Will correct and then merge.